supportedBackends/google/pipelines/v2beta/src/main/resources/gcs_transfer.sh

```diff
@@ -246,10 +245,8 @@ localize_directories() {
 
   if [[ ${USE_REQUESTER_PAYS} = true ]]; then
     rpflag="-u $project"
-  elif [[ ${USE_REQUESTER_PAYS} = false ]]; then
-    rpflag=""
   else
-    exit 1
+    rpflag=""
```

Comment:
not sure what the plan is wrt #5406 but if the work from there is being subsumed here then there are new commits to bring over

Reply:
a little bird told me

Reply:
I think I'll plan to rebase once you merge that PR